### PR TITLE
Fix error when virtual env is not set

### PR DESCRIPTION
**Pull request by maebert**
`os.getenv("_").endswith("pytest")` caused issues when the `_` env variable wasn't set (which makes this essentially `None.endswith("pytest")`, so let's not do this.

### CODE REVIEW

File: candles/sync_candles.py [hunk 1]
```diff
@@ -178,39 +178,9 @@ class SyncBitfinexCandles(BaseSyncCandles):
...
-        "p23",
-        "p24",
-        "p25",
-        "p26",
-        "p27",
-        "p28",
-        "p29",
-        "p30",
-    ]
+    PERIODS = [f"p{n}" for n in range(2, 31)]
```
Comment (maebert) [inline candles/sync_candles.py:183]: f-strings are my favourite thing about Python 3.6, and are roughly equivalent to `"p{n}".format(**locals())`

Comment (manos) [reply candles/sync_candles.py:183]: Nice - I remember reading about these but thought it was some version later than 3.6. Thanks!

**Comment by manos** [comment]
Interesting - so if "_" is not set, that implies pytest wasn't run from a bash-compatible shell, and the tests will be writing to the wrong db (candles.py:28). It will also set the ratelimit really low, to the prod value (candles_sync.py:181).
This was hacky to begin with and I'd love to find another way to detect "pytest is running" (note: their docs say pytest sets an ENV var, but it wasn't for me), and I think this PR will make tests a) write to our existing candles, b) slow.

**Comment by maebert** [comment]
Mmh, interesting. For me `os.getenv("_")` is always `None`, I thought that was related to using a virtual environment at first, because it just doesn't show up in python at all:

image

An easy way to test if pytest is running is `if "pytest" in sys.modules`. I'll change this PR accordingly.

**Comment by manos** [comment]
WFM, thanks!

**Pull request merged**